avahi-python/avahi-bookmarks.in

```diff
@@ -27,9 +27,14 @@ except ImportError:
     sys.exit(1)
 
 try:
-    import dbus.glib
-except ImportError:
-    pass
+    from dbus import DBusException
+    from dbus.mainloop.glib import DBusGMainLoop
+    DBusGMainLoop(set_as_default=True)
+except ImportError as e:
```

Comment:
I don't think you need the try/except for fallback to the old API: `DBusGMainLoop(set_as_default=True)` has worked since dbus-python 0.82.4, released in 2007.

Reply:
Okay, will trust you on this. That code is not a little obscure and not sure still used.